.github/ISSUE_TEMPLATE/bug_template.md

```diff
@@ -0,0 +1,66 @@
+---
+name: bug report
+about: use this template to report bugs
+title: ":bug: [BUG SUMMARY]"
+labels: bug, needs triage
+assignees: MiczFlor
+---
+
+## Hardware
+
+### RaspberryPi version
+
+i.e. `3 B+`
+
+### RFID Reader
+
+i.e. `16c0:27db HXGCoLtd Keyboard`
+
+### Soundcard
+
+i.e. `0d8c:0014 C-Media Electronics, Inc. Audio Adapter (Unitek Y-247A)`
+
+### Other notable hardware
+
+i.e.:
+```
+GPIO Buttons:
```

Comment:
Comment like "Please paste content from /home/pi/RPi-Jukebox-RFID/scripts/gpio_buttons.py here" would be helpful

Reply:
good idea.